action.yml

```diff
@@ -0,0 +1,24 @@
+name: Git Town - GitHub Action
+description: Visualizes your stacked changes when proposing pull requests on GitHub
```

Comment:
This description is important for finding this action, let's spend some time word-smithing it. What do you think about this:

```suggestion
description: Adds a map of the corresponding stacked change to pull requests
```

Reply:
Have moved this conversation to this discussion: https://github.com/git-town/action/discussions/3